frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SlideInUnmanagedBelowAction.java

```diff
@@ -0,0 +1,89 @@
+package com.virtuslab.gitmachete.frontend.actions.common;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
+import lombok.CustomLog;
+import lombok.experimental.ExtensionMethod;
+import lombok.val;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
+
+import com.virtuslab.branchlayout.api.BranchLayoutEntry;
+import com.virtuslab.gitmachete.frontend.actions.backgroundables.SlideInNonRootBackgroundable;
+import com.virtuslab.gitmachete.frontend.actions.backgroundables.SlideInRootBackgroundable;
+import com.virtuslab.gitmachete.frontend.actions.base.BaseGitMacheteRepositoryReadyAction;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeyGitMacheteRepository;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeySelectedBranchName;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeyUnmanagedBranchName;
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle;
+
+@ExtensionMethod(GitMacheteBundle.class)
+@CustomLog
+public class SlideInUnmanagedBelowAction extends BaseGitMacheteRepositoryReadyAction
```

Comment:
IntelliJ warns about action not registered in plugin.xml 🤔 Looks like a glitch in DevKit tho

Reply:
Looks that actions that aren't defined under a `<group>` are considered "unregistered in plugin.xml", 🐞 indeed